common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -417,6 +431,7 @@ public void run() {
                         commandCallbackOnError(command, result);
                         break;
                     case COMPLETED:
+                    case REFRESH:
```

Comment:
My understanding is `REFRESH` fails completed with success but still needs a forced refresh? Could there be a case when you need REFRESH but the request is completed successfully such that you can't invoke the `onTaskCompleted` callback and rather need to invoke the `onError` callback?

Reply:
So what is the Command Result returned back in that case?

Reply:
REFRESH_ON_ERROR

Reply:
Outdated